instrumentation/opentelemetry_cowboy/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
-## Unreleased
+## v0.2.0-beta.1
```

Comment:
Why beta.1?

Reply:
Rebar hex started griping that it won't allow "unstable" deps in stable releases.

Reply:
Oh, ugh, that needs to change for `0.x.y` versions.

Reply:
Or wait, doesn't it also complain if you have the `-beta`?

Either way, if updating to depending on 1.0.0 you shouldn't need the `beta.1`

Reply:
Yeah, it stopped griping when I made the lib a non-stable version.

It's griping because otel-telemetry is still in beta.

Reply:
I need to remove the auto-tracing stuff from it before I cut it to stable.

Reply:
Ah ok.

But still, 0.x aren't stable so it shouldn't complain either way.

Reply:
Yeah. Could be a bug. Here's what the output was.

```
 bryan@xxx  ~/dev/opentelemetry-erlang-contrib/instrumentation/opentelemetry_cowboy   otel-1.0 ±  rebar3 hex publish
===> Verifying dependencies...
===> The following pre-release dependencies were found :

opentelemetry_telemetry - 1.0.0-beta.7

In the future packages with pre-release dependencies will be considered unstable and will be prevented from being published. We recommend you upgrade your these dependencies as soon as possible

Publishing opentelemetry_cowboy 0.3.0 to hexpm
  Description: OpenTelemetry Cowboy Instrumentation
  Dependencies:
    telemetry ~> 1.0
    opentelemetry_telemetry ~> 1.0.0-beta.7
    opentelemetry_api ~> 1.0
    cowboy_telemetry ~> 0.4
  Included files:
    CHANGELOG.md
    LICENSE
    README.md
    rebar.config
    rebar.lock
    src/opentelemetry_cowboy.app.src
    src/opentelemetry_cowboy.erl
  Licenses: Apache 2.0
  Links:
    GitHub: https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/instrumentation/opentelemetry_cowboy
  Build tools: rebar3
Be aware, you are publishing to the public Hexpm repository.
Before publishing, please read Hex CoC: https://hex.pm/policies/codeofconduct
Proceed? ("Y")> y
Local Password:
===> Failed to publish package: Validation error(s)
	Requirements could not be computed
opentelemetry_telemetry
--------------------
invalid requirement: "~> 1.0.0-beta.7", unstable requirements are not allowed for stable releases
```

Reply:
Oh crap, that is a response from hex.pm... I think I already complained about this and didn't succeed :)